requirements.txt

```
@@ -20,6 +20,7 @@ dynaconf>=3.1.9,<=3.1.11
 gunicorn>=20.1,<=20.1.0
 jinja2>=3.1,<=3.1.2
 naya>=1.1.1,<=1.1.1
+pulp-glue>=0.17.1,<1.0
```

There is no such guarantee.
```suggestion
pulp-glue>=0.17.1,<0.18
```